.github/workflows/ci.yml

```diff
@@ -5,26 +5,44 @@ on: [push, pull_request]
 
 jobs:
   test:
+    name: Run workflow tests
     runs-on: ubuntu-latest
     strategy:
       matrix:
         nxf_ver: ['19.10.0', '']
         aligner: ["--aligner 'hisat2'", "--aligner 'star'", "--pseudo_aligner 'salmon'"]
         options: ['--skipQC', '--remove_rRNA', '--saveUnaligned', '--skipTrimming', '--star_index false']
     steps:
-      - uses: actions/checkout@v2
+      - name: Check out pipeline code
+        uses: actions/checkout@v2
+
+      - name: Check if Docker / Conda changed
+        uses: technote-space/get-diff-action@v1
+        with:
+          PREFIX_FILTER: |
+            Dockerfile
+            environment.yml
+
+      - name: Build new docker image
+        if: env.GIT_DIFF
+        run: docker build . -t nfcore/rnaseq:dev
+
+      - name: Pull docker image
+        if: ${{ !env.GIT_DIFF }}
+        run: |
+          docker pull nfcore/rnaseq:dev
+          docker tag nfcore/rnaseq:dev nfcore/rnaseq:dev
```

Comment:
```suggestion
        run: docker pull nfcore/rnaseq:dev
```

Reply:
This is what I meant ;-)

Reply:
Yup, I understand - see https://github.com/nf-core/rnaseq/pull/423#discussion_r433910287 for why it wouldn't work.